modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-rollback_{context}"]
+= Rolling back the network provider to OpenShift SDN
+
+As a cluster administrator, you can rollback the Pod network provider for your cluster to the OpenShift SDN Pod network provider.
+During the rollback, you must reboot every node in your cluster.
+
+[IMPORTANT]
+====
+Only rollback to OpenShift SDN if the migration to the OVN-Kubernetes Pod network provider is unsuccessful.
+====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on bare metal infrastructure configured with the OVN-Kubernetes Pod network provider.
+
+. The OpenShift SDN network provider namespace must be deleted prior to the rollback. To remove the namespace, enter the following command:
```

Comment:
We don't need to remove the namespace prior to the rollback. We shall ask users to do that if they don't need a rollback.